src/index.html

```diff
@@ -9,6 +9,21 @@
         <meta name="keywords" content="PLY, Gaussian, Splat, PlayCanvas, Editor" />
         <meta name="viewport" content="width=device-width, user-scalable=no, minimum-scale=1.0, maximum-scale=1.0" />
         <link rel="stylesheet" type="text/css" href="style.css">
+
+        <!-- Open Graph / Facebook -->
+        <meta property="og:type" content="website" />
+        <meta property="og:url" content="https://playcanvas.com/super-splat" />
+        <meta property="og:title" content="SuperSplat" />
+        <meta property="og:description" content="SuperSplat is an advanced browser-based editor for manipulating and optimizing 3D Gaussian Splats. It is open source and engine agnostic.." />
```

Comment:
```suggestion
        <meta property="og:description" content="SuperSplat is an advanced browser-based editor for manipulating and optimizing 3D Gaussian Splats. It is open source and engine agnostic." />
```

Reply:
Looks like this was copy-pasted from the meta description - there are two full stops there too. Please remove that extra one too!